### PR TITLE
Added new instructor modal.

### PR DESCRIPTION
**Pull request by waheedahmed**
[ECOM-6391](https://openedx.atlassian.net/browse/ECOM-6391)

Added frontend design behind feature flag, by default hidden.

screen shot 2017-01-19 at 7 56 20 pm
screen shot 2017-01-20 at 12 44 39 pm

### CODE REVIEW

**Comment by waheedahmed** [comment]
@bderusha, @awais786, @tasawernawaz  Please review.

File: course_discovery/apps/publisher/migrations/0028_create_switch_hide_add_instructor_feature.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from django.db import migrations
+
+
+def create_switch(apps, schema_editor):
+    """Create the publisher_hide_add_instructor_feature switch if it does not already exist."""
+    Switch = apps.get_model('waffle', 'Switch')
+    Switch.objects.get_or_create(name='publisher_hide_add_instructor_feature', defaults={'active': True})
```
Comment (bderusha) [inline course_discovery/apps/publisher/migrations/0028_create_switch_hide_add_instructor_feature.py]: In general I'd prefer if we named our feature flags in an affirmative way

`publisher_add_instructor_feature`

File: course_discovery/apps/publisher/views.py [hunk 2]
```diff
@@ -191,7 +191,8 @@ def get_context_data(self):
             'course_form': self.course_form,
             'run_form': self.run_form,
             'seat_form': self.seat_form,
-            'publisher_hide_features_for_pilot': waffle.switch_is_active('publisher_hide_features_for_pilot')
+            'publisher_hide_features_for_pilot': waffle.switch_is_active('publisher_hide_features_for_pilot'),
+            'publisher_hide_add_instructor_feature': waffle.switch_is_active('publisher_hide_add_instructor_feature')
```
Comment (bderusha) [inline course_discovery/apps/publisher/views.py]: NIT: add comma at end of line

**Review by bderusha — APPROVED** [review]
👍 once comments addressed

**Review by tasawernawaz — APPROVED** [review]

**Pull request merged**